collection-scripts/gather_ceph_logs

```diff
@@ -17,6 +17,17 @@ for ns in $namespaces; do
         oc rsync -n "${ns}" "$(oc get pods -n "${ns}" --no-headers | grep "${node//./}-debug" | awk '{print $1}')":/host/var/lib/rook/"${ns}"/log "${CEPH_DAEMON_LOG_OUTPUT_DIR}"
     }
 
+    # Collecting csi and ceph daemon logs
+    ceph_daemon_log_collection() {
+        dbglog "collecting csi and ceph daemon logs from node ${node}"
+        # as per the correction we are keeping both /var/lib/rook & /var/log/ceph for ceph pods, as to not break anything
+        oc rsync -n "${ns}" "$(oc get pods -n "${ns}" --no-headers | grep "${node//./}-debug" | awk '{print $1}')":/host/var/log/ceph/ "${CEPH_DAEMON_LOG_OUTPUT_DIR}"
```

Comment:
once the csi operator changes are done we need to add the query the driver and get the host path here.

Reply:
ceph daemon logs are collected above right?

Reply:
The path is incorrect `/var/lib/rook` which should be `/var/log/ceph`

As discussed offline already we will keep both as not break anything

Reply:
This is resolved then, right?

Reply:
Yes correct, will have a follow up PR once CSI-operator is there, 
Till now it works with Rook operator